Sources/CoreCommands/SwiftTool.swift

```diff
@@ -675,6 +676,7 @@ public final class SwiftTool {
             dataPath: dataPath,
             configuration: options.build.configuration,
             toolchain: toolchain,
+            hostTriple: hostTriple,
```

Comment:
Again avoiding recompute because we already know the triple from `getHostToolchain()`.